Sources/Orbit/Support/Environment Keys/TextColorKey.swift

```diff
@@ -0,0 +1,27 @@
+import SwiftUI
+
+struct TextAccentColorKey: EnvironmentKey {
```

Comment:
This isn't `iconColor` - it's added in the next commit instead of this one. The commit message a is a bit confusing now.

Reply:
🤦 thx, rebased and updated